server/api/controllers/admin.js

```diff
@@ -113,7 +113,7 @@ const putAdminKit = (req, res) => {
 	}
 
 	const auditInfo = { userEmail: req?.auth?.sub?.email, sessionId: req?.session_id, apiPath: req?.swagger?.apiPath, method: req?.swagger?.operationPath?.[2] };
-	toolsLib.updateKitConfigSecrets(data, req.auth.scopes, auditInfo)
+	toolsLib.updateKitConfigSecrets(data, req.auth.scopes, auditInfo, req.auth?.sub?.permissions, req.auth.sub.id)
```

Comment:
whats this for?

Reply:
this is for the kit update endpoint. for managing permisssions on each field of the config data